language/stdlib/modules/libra_time.mvir

```diff
@@ -0,0 +1,28 @@
+module LibraTime {
```

Comment:
To your top-level comment: I think we perhaps make this immutable and then call it `Timestamp` or `LibraTimestamp`. A mutable representation of a timestamp seems certain to cause confusion...